packages/manager/apps/pci-billing/src/components/TabsPanel.component.tsx

```diff
@@ -0,0 +1,75 @@
+import { useNotifications } from '@ovh-ux/manager-react-components';
+import { ShellContext } from '@ovh-ux/manager-react-shell-client';
+import {
+  OsdsTabBar,
+  OsdsTabBarItem,
+  OsdsTabs,
+} from '@ovhcloud/ods-components/react';
+import React, { useContext, useEffect, useState } from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+
+export type TabItemProps = {
+  name: string;
+  title: string | JSX.Element;
+  to?: string;
+  tracking?: string;
+  isDisabled?: boolean;
+};
+
+export type TabsProps = {
+  tabs: TabItemProps[];
+};
+
+export default function TabsPanel({ tabs }: Readonly<TabsProps>) {
+  const [activePanel, setActivePanel] = useState('');
+  const location = useLocation();
+  const { clearNotifications } = useNotifications();
+  const { tracking } = useContext(ShellContext)?.shell || {};
```

Comment:
```suggestion
  import { useTracking } from '@ovh-ux/manager-react-shell-client';
```

This way you can mock it while uni testing without having to mock useContext(i got the case)

Reply:
it's deprecated https://github.com/ovh/manager/blob/master/packages/manager/core/shell-client/src/hooks/useTracking.ts#L5